### ISSUE TITLE
Satellite queue naming

### ISSUE THREAD

**Issue by SeanFeldman** [Particular/NServiceBus#4451]
While reviewing a bug fix for https://github.com/Particular/NServiceBus.AzureServiceBus/issues/480 for ASB, have noticed that in order to create a custom satellite, we don't allow an easy way to follow NSB convention for satellite naming. 

Our satellite registration API ([sample](https://docs.particular.net/nservicebus/satellites/)) expects a transport address. It doesn't allow to pass in a qualifier and let the FW handle the transport address construction. Should one want to follow the convention, they'd need to know the ["procedure" to turn a qualifier into a transport address](https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs#L67-L68).

Should we provide an overload that takes a satellite `qualifier` and generates a transport address using NSB satellite conventions w/o burden of the conversion mentioned above?

**Comment by SeanFeldman** [comment]
cc @bording

**Comment by timbussmann** [comment]
yeah, the whole endpoint address stuff is a nightmare (also for routing). The qualifier idea would definitely work, maybe this is a reasonable enough workaround but I'd prefer to fix this once for all.

**Comment by andreasohlund** [comment]
Me and @timbussmann  decided to park this on the future milestone for now since there is a workaround and we have other more pressing issues atm

**Comment by timbussmann** [comment]
btw. this is related imho: https://github.com/Particular/NServiceBus/issues/4111
